tests/conftest.py

```diff
@@ -16,17 +17,27 @@ def is_responsive(url):
         return False
 
 
+@pytest.fixture(scope="session")
+def screenshot_dir():
+    sdir = Path.joinpath(Path.cwd(), "screenshots")
+    try:
+        os.mkdir(sdir)
+    except FileExistsError:
+        pass
+    return sdir
+
+
 @pytest.fixture(scope="session")
 def notebook_service(docker_ip, docker_services):
     """Ensure that HTTP service is up and responsive."""
 
     docker_compose = docker_services._docker_compose
 
     # assurance for host user UID other that 1000
-    chown_command = "exec -T -u root aiidalab bash -c 'chown -R jovyan:users /home/jovyan/apps/aiidalab-widgets-base'"
+    chown_command = "exec -T -u root aiidalab bash -c 'chmod -R a+rw /home/jovyan/apps/aiidalab-widgets-base'"
```

Comment:
```suggestion
    chmod_command = "exec -T -u root aiidalab bash -c 'chmod -R a+rw /home/jovyan/apps/aiidalab-widgets-base'"
```

Reply:
nitpick. We have `aiidalab_exec` soon by https://github.com/aiidalab/aiidalab-widgets-base/pull/409

Reply:
Yep, I am aware, I would like to merge this first so that we have screenshots in that PR. You can then create multiple screenshots alo the various stages of the new test.